### PR TITLE
Random data and some cleanups

### PR DESCRIPTION
**Pull request by iostream72**
* Add a randomized data generator class
* New class VentilatorData which encapsulates the ventilator data
* Add ability to configure per environment via a yaml file
* Add a dev startup script (start_dev.sh)

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -1,46 +1,41 @@
...
-    def __init__(self, **kwargs):
-        self.serial_connection = kwargs['serial_connection']
 
-    def get(self):
-        line = self.serial_connection.request("getStats\n")
-        print(line)
-        json_line = json.loads(line)
-        print(json_line)
-        return json_line
+@app.before_first_request
```
Comment (CaptainGlac1er) [inline server.py]: Could you adapt this to how it is currently on my branch?

File: start.sh [hunk 2]
```diff
@@ -1,2 +1,2 @@
-# wsgi:create_app(config="CONSOLE") if you want to run on desktop
-gunicorn --bind 0.0.0.0:8080 'wsgi:create_app()'
+#!/usr/bin/env sh
+sudo gunicorn --bind 0.0.0.0:80 wsgi:app
```
Comment (CaptainGlac1er) [inline start.sh]: revert this line to `gunicorn --bind 0.0.0.0:8080 'wsgi:create_app()'`

File: README.md [hunk 3]
```diff
@@ -51,8 +51,12 @@ Eventually we'll have pre-deployed images ready for download and quick install o
...
 17. After you connect your properly programmed Arduino via a USB cable, you should be able to visit `http://ventilator-1.local/api/ventilator` and view JSON result of the latest ventilator stats. Refresh the page to query the Arduino again and get different values.
 
-### How to test using dev mode
-Execute wsgi.py with the flag `--dev` 
-Sample return data `{"tidalVolume" : "500","respiratoryRate" : "25","peakInspiratoryPressure" : "70","ieRatio" : "1:3","peep" : "7"}`
+### Local Simulation
+
+If you would like to run the web UI and hit a local simulation interface, make sure you have
+all the dependencies installed:
+1. `pip install flask flask-cors pyserial pyyaml`
```
Comment (CaptainGlac1er) [inline README.md]: pipenv install would make sure the user downloads the right versions as well.

Comment (iostream72) [reply README.md]: Added

**Pull request merged**